Lib/test/test_asyncio/test_windows_utils.py

```diff
@@ -107,7 +107,8 @@ def test_popen(self):
 
         events = [ovin.event, ovout.event, overr.event]
         # Super-long timeout for slow buildbots.
-        res = _winapi.WaitForMultipleObjects(events, True, 10000)
+        res = _winapi.WaitForMultipleObjects(events, True,
+                                             support.SHORT_TIMEOUT * 1000)
```

Comment:
> TypeError: WaitForMultipleObjects() argument 3 must be int, not float

support.SHORT_TIMEOUT looks like float value (30.0)

Reply:
Oh right, it should now be fixed.